src/sidebar/services/annotations-exporter.tsx

```diff
@@ -91,7 +91,7 @@ export class AnnotationsExporter {
       const page = pageLabel(annotation);
       const annotationQuote = quote(annotation);
       const lines = [
-        `Created at: ${formatDateTime(new Date(annotation.created))}`,
+        `Created at: ${formatSortableDateTime(new Date(annotation.created))}`,
```

Comment:
Can you remind me why we needed to use a date/time format that has this lexicograph / chronological sorting equivalence?

https://github.com/hypothesis/client/pull/6079 was the PR that introduced the change, but the only rationale in that PR was to have a more human-friendly form of the ISO date.